applications/train.py

```diff
@@ -359,16 +408,21 @@ def trainer(conf, trial=False, verbose=True):
     # Train and validate
     results_dict = defaultdict(list)
     
+    # folder to save trial items
+    os.makedirs(save_loc+"/trial"+str(int(trial.number)), exist_ok=True)
+    
     for epoch in list(range(epochs)):
         
         t_loss, t_corr, t_true = train_one_epoch(
             model, train_loader, optimizer, train_loss, nc
         )
         v_loss, v_corr, v_true, metrics, cesm_metrics = validate(
-            model, valid_loader, valid_loss, validation_metrics, validation_metrics_cesm, nc
+            model, valid_loader, valid_loss, validation_metrics, validation_metrics_cesm, 
```

Comment:
Similar to the previous comment. Should we include a way of making the plotting just for some epochs/trials? Could it be possible that this generates "too many" images?

Reply:
Agreed! See comment above -- I am running a study now and it is A LOT of images...